src/SIM/AdaptiveSIM.C

```diff
@@ -310,9 +318,17 @@ bool AdaptiveSIM::adaptMesh (int iStep)
 
   if (threshold == TRUE_BETA)
   {
+    if (model.getNoPatches() > 1) {
+      std::cerr << "True beta refinement not available for multi-patch models\n";
+      return false;
+    }
     IFEM::cout <<"\nRefining by increasing solution space by "<< beta
                <<" percent."<< std::endl;
-    prm.errors = eNorm.getRow(eRow);
+    Vector errors(model.getPatch(1)->getNoNodes(1));
+    static_cast<ASMunstruct*>(model.getPatch(1))->remapErrors(errors, eNorm.getRow(eRow));
+    prm.errors.reserve(errors.size());
+    for (auto &it : errors)
+      prm.errors.push_back(it);
```

Comment:
Looks unnecessary with the loop, why not just `prm.errors = errors`', or even better, pass prm.errors directly to remapError and avoid the copy all together.

Reply:
sorry cleaned SIMinput missed this one. ASMuxD.h is removed, ASMunstruct is kept and the LR includes are now gone. the loop was a leftover from the old code using a different data structure and has been changed to remapping directly